packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig

```diff
@@ -1,12 +1,12 @@
 {% set accordion = actionStep.accordion %}
 {% set suffix = actionStep.suffix %}
+{% set link = actionStep.link %}
 
 <section
   class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
   {% if accordion %}
     <button
-      class="ma__action-step__header js-accordion-link"
-      aria-label="{{ actionStep.accordionLabel }}">
+      class="ma__action-step__header js-accordion-link">

```

Comment:
```suggestion
      class="ma__action-step__header js-accordion-link"
      {{ actionStep.accordionLabel ? aria-label="{{ actionStep.accordionLabel }}" : '' }}>
```
Instead of removing this can you make it conditional? This is probably the wrong syntax ^

Reply:
Since I'm not sure if other uses need this field.